#Preprints

Jones, A., Townes, W. F., Li, D., & Engelhardt, B. E.. (2022). Alignment of spatial genomics and histology data using deep Gaussian processes. [PDF]
Gewirtz, A. D., Townes, W. F., & Engelhardt, B. E.. (2021). Telescoping bimodal latent Dirichlet allocation to identify expression QTLs across tissues. Cold spring harbor laboratory. [PDF]
Dumitrascu, B., Feng, K., & Engelhardt, B. E.. GT-TS: Experimental design for maximizing cell type discovery in single-cell data. [PDF]
Chaney, A. J. B., Verma, A., Lee, Y., & Engelhardt, B. E.. Nonparametric deconvolution models. [PDF]
Gao, C., Brown, C. D., & Engelhardt, B. E.. A latent factor model with a mixture of sparse and dense factors to model gene expression data affected by technical and biological covariates. [PDF]
Verma, A., & Engelhardt, B. E.. A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments. [PDF]
Grabski, I. N., de Vito, R., & Engelhardt, B. E.. Bayesian ordinal quantile regression with a partially collapsed Gibbs sampler. [PDF]
Mandyam, A., Jones, A., Laudanski, K., & Engelhardt, B. E.. Nested policy reinforcement learning. [PDF]
Li*, D., Jones*, A., & Engelhardt, B. E.. Probabilistic contrastive principal component analysis. [PDF]
Townes, W. F., & Engelhardt, B. E.. Nonnegative spatial factorization. [PDF]
de Vito, R., Grabski, I. N., Aguiar, D., Schneper, L. M., Verma, A., {Castillo Fernandez}, J., Mitchell, C., Bell, J., McLanahan, S., Notterman, D. A., & Engelhardt, B. E.. Differentially methylated regions and methylation QTLs for teen depression and early puberty in the Fragile Families Child Wellbeing Study. [PDF]
