---
title: "Publications"
permalink: /publications/
date: 2022-06-11
---

## Preprints

AD Gewirtz, FW Townes, BE Engelhardt (2022). Expression QTLs in single-cell sequencing data. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.08.14.503915v1.abstract)] 

A Jones, G Gundersen, BE Engelhardt (2022). Linking histology and molecular state across human tissues. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.06.10.495669v1)]

T Fitzgerald, A Jones, BE Engelhardt (2022). A Poisson reduced-rank regression model for association mapping in sequencing data. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.05.31.494236)] [[Code](https://github.com/tianafitz/PRRR)]

A Mandyam, A Jones, K Laudanski, BE Engelhardt. Nested policy reinforcement learning. [[arXiv](https://arxiv.org/abs/2110.02879)]

A Jones, FW Townes, D Li, BE Engelhardt (2022). Alignment of spatial genomics and histology data using deep Gaussian processes. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.01.10.475692)] [[Code](https://github.com/andrewcharlesjones/spatial-alignment)] [[Post](https://stanfordbeehive.github.io/GPSA/)]

D Li\*, A Jones\*, BE Engelhardt. Probabilistic contrastive principal component analysis. [[arXiv](https://arxiv.org/abs/2012.07977)] [[Code](https://github.com/andrewcharlesjones/pcpca)]

D Li\*, A Jones\*, S Banerjee, BE Engelhardt. Multi-group Gaussian processes. [[arXiv](https://arxiv.org/abs/2110.08411)] [[Code](https://github.com/andrewcharlesjones/multi-group-GP)]

IN Grabski, R de Vito, BE Engelhardt. Bayesian ordinal quantile regression with a partially collapsed Gibbs sampler. [[arXiv](https://arxiv.org/abs/1911.07099)] [[Code](https://github.com/igrabski/boqr_bal)]

A Verma, BE Engelhardt. A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.14.906313)] [[Code](https://github.com/architverma1/sc-manifold-alignment)]

R de Vito, IN Grabski, D Aguiar, LM Schneper, A Verma, J Castillo Fernandez, C Mitchell, JT Bell, S McLanahan, DA Notterman, BE Engelhardt. Differentially methylated regions and methylation QTLs for teen depression and early puberty in the Fragile Families Child Wellbeing Study. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2021.05.20.444959v1)]

B Dumitrascu, K Feng, BE Engelhardt. GT-TS: Experimental design for maximizing cell type discovery in single-cell data. [[bioRxiv](https://www.biorxiv.org/content/10.1101/386540)] [[Talk](https://www.youtube.com/watch?v=sDFe3e33m_I)]

AJB Chaney, A Verma, Y Lee, BE Engelhardt. Nonparametric deconvolution models. [[arXiv](arxiv.org/abs/2003.07718)] [[Code](https://github.com/ajbc/ndm)]

C Gao, CD Brown, BE Engelhardt. A latent factor model with a mixture of sparse and dense factors to model gene expression data affected by technical and biological covariates. [[arXiv](https://arxiv.org/abs/1310.4792)]

## Publications

FW Townes, BE Engelhardt (2022). Nonnegative spatial factorization. *Nature Methods (accepted)* [[arXiv](https://arxiv.org/abs/2110.06122)] [[Code](https://github.com/willtownes/nsf-paper)]

AD Gewirtz, FW Townes, BE Engelhardt (2022). Telescoping bimodal latent Dirichlet allocation to identify expression QTLs across tissues. *Life Scienec Alliance (to appear)*. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2021.10.27.466156)] [[Code](https://github.com/gewirtz/TBLDA)] [[Post](https://stanfordbeehive.github.io/TBLDA/)]

SG Jena, AG Goglia, BE Engelhardt (2022). Towards ‘end-to-end’ analysis and understanding of biological timecourse data. *Biochemical Journal* 479(11): 1257–1263. [[PDF](https://doi.org/10.1042/BCJ20220053)]

G Martinet, A Strzalkowski, BE Engelhardt (2022). Variance minimization in the Wasserstein space for invariant causal prediction. *Artificial Intelligence and Statistics (AISTATS)*. [[PDF](https://proceedings.mlr.press/v151/martinet22a/martinet22a.pdf)] [[Code](https://github.com/astrzalk/WVM_reproducibility)] [[Post](https://stanfordbeehive.github.io/WMD/)]

A Jones, FW Townes, D Li, BE Engelhardt (2022). Contrastive latent variable modeling with application to case-control sequencing experiments. *Annals of Applied Statistics (AOAS)*. [[PDF](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-16/issue-3/Contrastive-latent-variable-modeling-with-application-to-case-control-sequencing/10.1214/21-AOAS1534.short)] [[arXiv](https://arxiv.org/abs/2102.06731)] [[Code](https://github.com/andrewcharlesjones/cplvm)]

S Cui, EC Yoo, D Li, K Laudanski, BE Engelhardt (2022). Hierarchical Gaussian processes and mixtures of experts to model Covid-19 patient trajectories. *Proceedings of the Pacific Symposium on Biocomputing (PSB)*. [[PDF](https://dl.acm.org/doi/pdf/10.1145/3459930.3469536)] [[Code](https://github.com/eyeshoe/cop-e-cat)]

J Ash\*, G Darnell\*, D Munro\*, BE Engelhardt (2021). Joint analysis of gene expression levels and histological images identifies genes associated with tissue morphology. *Nature Communications* 12(1609). [[PDF](https://www.nature.com/articles/s41467-021-21727-x)] [[Code](https://github.com/daniel-munro/imageCCA)]

B Dumitrascu, S Villar, DG Mixon, BE Engelhardt (2021). Optimal marker gene selection for cell type discrimination in single cell analyses. *Nature Communications* 12(1186). [[PDF](https://www.nature.com/articles/s41467-021-21453-4)] [[Code](https://github.com/solevillar/scGeneFit-python)]

J Lu, B Dumitrascu, IC McDowell, AK Barrera, SM Leichter, TE Reddy, BE Engelhardt (2021). Causal network inference from gene transcriptional time series response to glucocorticoids. *PLoS Computational Biology* 17(1):e1008223.
[[PDF](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008223)] [[Code](https://github.com/lujonathanh/BETS)]

GW Gundersen, D Cai, C Zhou, BE Engelhardt, RP Adams (2021). Active multi-fidelity Bayesian online changepoint detection. *Uncertainty in Artifical Intelligence (UAI).* [[PDF](https://proceedings.mlr.press/v161/gundersen21a/gundersen21a.pdf)] [[Code](https://github.com/gwgundersen/bocd)]

A Wu, SA Nastase, CA Baldassano, NB Turk-Browne, KA Norman, BE Engelhardt, JW & Pillow (2021). Brain kernel: A new spatial covariance function for fMRI data. *NeuroImage* 15(245):118580. [[PDF](https://www.sciencedirect.com/science/article/pii/S1053811921008533?via%3Dihub)]

A Verma, SG Jena, DR Isakov, K Aoki, JE Toettcher, BE Engelhardt (2021). A self-exciting point process to study multi-cellular spatial signaling patterns. *Proceedings of the National Academy of Sciences (PNAS)* 118:e2026123118. [[PDF](https://www.pnas.org/doi/10.1073/pnas.2026123118)] [[Code](https://github.com/architverma1/CPP)]

A Mandyam, EC Yoo, J Soules, K Laudanski, BE Engelhardt (2021). COP-E-CAT: Cleaning and organization pipeline for EHR computational and analytic tasks. *Proceedings of the 12th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (BCB)*. [[PDF](https://dl.acm.org/doi/pdf/10.1145/3459930.3469536)] [[Code](https://github.com/aishwarya-rm/cop-e-cat)]

GW Gundersen, MW Zhang, BE Engelhardt (2021). Latent variable modeling with random features. *Artificial Intelligence and Statistics (AISTATS).*
[[PDF](http://proceedings.mlr.press/v130/gundersen21a/gundersen21a.pdf)] [[Code](https://github.com/gwgundersen/rflvm)]

N Prasad, BE Engelhardt, F Doshi-Velez (2020). Defining admissible rewards for high confidence policy evaluation. *ACM Conference on Health, Inference, and Learning (CHIL)*. [[PDF](https://dl.acm.org/doi/10.1145/3368555.3384450)]

L-F Cheng, B Dumitrascu, MM Zhang, C Chivers, ME Draugelis, K Li, BE Engelhardt (2020). Patient-specific effects of medication using latent force models with Gaussian processes. *Artificial Intelligence and Statistics (AISTATS).* [[PDF](https://proceedings.mlr.press/v108/cheng20c.html)]

M Salganik et al. (2020). Measuring the predictability of life outcomes with a scientific mass collaboration. *Proceedings of the National Academy of Sciences (PNAS)* 117(15):8398-8403. [[PDF](https://www.pnas.org/doi/10.1073/pnas.1915006117)]

L Cheng, G Darnell, C Chivers, ME Draugelis, K Li, BE Engelhardt (2020). Sparse multi-output Gaussian processes for medical time series prediction. *BMC Medical Informatics and Decision Making* 20(152). [[PDF](https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/s12911-020-1069-4)] [[Code](https://github.com/bee-hive/MedGP)]

F Camerlenghi\*, B Dumitrascu\*, F Ferrari, BE Engelhardt, S Favaro (2020). Nonparametric Bayesian multi-armed bandits for single cell experiment design. *Annals of Applied Statistics (AOAS)* 14(4):2003-2019. [[PDF](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-14/issue-4/Nonparametric-Bayesian-multiarmed-bandits-for-single-cell-experiment-design/10.1214/20-AOAS1370.short)] [[Code](https://github.com/fedfer/HPYsinglecell)]

M Oliva, GTEx Consortium, et al. (2020). The impact of sex on gene expression across human tissues. *Science* 369(6509) 
[[PDF](https://www.science.org/doi/10.1126/science.aba3066)]

GTEx Consortium. (2020). The GTEx Consortium atlas of genetic regulatory effects across human tissues. *Science* 369:1318–1330. 
[[PDF](https://www.science.org/doi/10.1126/science.aaz1776)]

D Gill, M Arvanitis, P Carter, AHI Cordero, B Jo, V Karhunen, SC Larsson, X Li, SM Lockhart, A Mason, E Pashos, A Saha, VY Tan, V Zuber, Y Bossé, S Fahle, K Hao, T Jiang, P Joubert, AC Lunt†, WH Ouwehand, DJ Roberts, W Timens, M van den Berge, NA Watkins, A Battle, AS Butterworth, J Danesh, ED Angelantonio, BE Engelhardt, JE Peters, DD Sin, S Burgess (2020). ACE inhibition and cardiometabolic risk factors, lung *ACE2* and *TMPRSS2* gene expression, and plasma *ACE2* levels: a Mendelian randomization study. *Royal Society of Open Science* 7:200958. [[PDF](https://pubmed.ncbi.nlm.nih.gov/33391794/)]

A Verma, BE Engelhardt (2020). A robust nonlinear low-dimensional manifold for single cell RNA-seq data. *BMC Bioinformatics* 21(324).
[[PDF](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03625-z)] [[Code]](https://github.com/architverma1/tGPLVM)

R Elyanow, B Dumitrascu, BE Engelhardt, BE Raphael (2020). netNMF-sc: Leveraging gene-gene interactions for imputation and dimensionality reduction in single-cell expression analysis. *Genome Research*, 20(2):195–204. [[PDF](https://genome.cshlp.org/content/early/2020/01/28/gr.251603.119)]

L Cheng, N Prasad, BE Engelhardt (2019). An optimal policy for patient laboratory tests in intensive care units. *Proceedings of the Pacific Symposium on Biocomputing (PSB)*. [[PDF](https://www.worldscientific.com/doi/10.1142/9789813279827_0029)]

G Guan, BE Engelhardt (2019). Predicting sick patient volume in a pediatric outpatient setting using time series analysis. *Proceedings of Machine Learning for Health Care (MLHC).* [[PDF](http://proceedings.mlr.press/v106/guan19a/guan19a.pdf)]

G Gundersen, B Dumitrascu, BE Engelhardt (2019). End-to-end training of deep probabilistic CCA on paired biomedical observations. *Proceedings of the Conference on Uncertainty in Artificial Intelligence (UAI).*
[[PDF](http://proceedings.mlr.press/v115/gundersen20a.html)]

IC McDowell, D Manandhar, CM Vockley, A Schmid, TE Reddy\*, BE Engelhardt\* (2018). Clustering gene expression time series data using an infinite Gaussian process mixture model. *PLoS Computational Biology*, 14(1):e1005896.
[[PDF](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005896)] [[Code](https://github.com/PrincetonUniversity/DP_GP_cluster)]

B Dumitrascu, G Darnell, J Ayroles, BE Engelhardt (2018). A Bayesian test to identify variance effects. *Bioinformatics*, bty565.
[[PDF](https://academic.oup.com/bioinformatics/article/35/2/200/5050024)] [[Code]](https://github.com/b2du/bth)

IC McDowell, A Barrero, AM D’Ippolito, CM Vockley, LK Hong, SM Leichter, LC Bartelt, WH Majoros, L Song, A Safi, DD Koçak, CA Gersbach, AJ Hartemink, GE Crawford, BE Engelhardt, TE Reddy (2018). Glucocorticoid receptor recruits to enhancers and drives activation by motif-directed binding. *Genome Research*, 28(9):1272-1284.
[[PDF](https://pubmed.ncbi.nlm.nih.gov/30097539/)]

D Aguiar, L Cheng, B Dumitrascu, F Mordelet, AA Pai, BE Engelhardt (2018). BIISQ: Bayesian nonparametric discovery of Isoforms and Individual Specific Quantification. *Nature Communications* 9(1681). [[PDF](https://www.nature.com/articles/s41467-018-03402-w)] [[Code](https://github.com/bee-hive/BIISQ)]

AJB Chaney, BM Stewart, BE Engelhardt (2018). How algorithmic confounding in recommendation systems increases homogeneity and decreases utility. *Proceedings of the 12th ACM Conference on Recommender Systems.* [[PDF](https://dl.acm.org/doi/10.1145/3240323.3240370)]

B Dumitrascu*, K Feng*, BE Engelhardt (2018). PG-TS: Improved Thompson sampling for logistic contextual bandits. *Proceedings of Neural Information Processing Systems (NeurIPS)*. [[PDF](https://proceedings.neurips.cc/paper/2018/file/ce6c92303f38d297e263c7180f03d402-Paper.pdf)]

JD Cohen, N Daw, BE Engelhardt, U Hasson, K Li, Y Niv, KA Norman, J Pillow, PJ Ramadge, NB Turk-Browne, TL Willke (2017). Computational approaches to fMRI analysis. *Nature Neuroscience* 20(3):304–313. [[PDF](https://www.nature.com/articles/nn.4499)]

S Srivastava, BE Engelhardt, DB Dunson (2017). Expandable factor analysis. *Biometrika* 104(3):649–663.[[PDF](https://academic.oup.com/biomet/article/104/3/649/3869470)]

GTEx Consortium, A Battle\*, CD Brown\*, BE Engelhardt\*, SM Montgomery\* (2017). Genetic effects on gene expression across human tissues. *Nature* 550: 204–213. [[PDF](https://www.nature.com/articles/nature24277)]

S Zhao, BE Engelhardt, S Mukherjee, DB Dunson (2017). Fast moment estimation for generalized latent Dirichlet models. *Journal of the American Statistical Association (JASA)* 113(524):1528–1540. [[PDF](https://www.tandfonline.com/doi/full/10.1080/01621459.2017.1341839)]

G Darnell, S Georgiev, S Mukherjee, BE Engelhardt (2017). Adaptive randomized dimension reduction on massive data. *Journal of Machine Learning Research (JMLR)* 18(140):1-30. [[PDF](https://jmlr.org/papers/v18/15-143.html)]

A Saha, Y Kim*, ADH Gewirtz*, B Jo, C Gao, IC McDowell, GTEx Consortium, BE Engelhardt\*, A Battle\* (2017). Co-expression networks reveal the tissue-specific regulation of transcription and splicing. *Genome Research* 27(11):1843–1858. [[PDF](https://genome.cshlp.org/content/27/11/1843.short)]

N Prasad, L-F Cheng, C Chivers, M Draugelis, BE Engelhardt (2017). A reinforcement learning approach to weaning of mechanical ventilation in intensive care units. *Proceedings of Uncertainty in Artificial Intelligence (UAI)*. [[PDF](https://auai.org/uai2017/proceedings/papers/209.pdf)]

G Jerfel, ME Basbug, BE Engelhardt (2017). Dynamic collaborative filtering with compound Poisson factorization. *Proceedings of Artificial Intelligence and Statistics (AISTATS)* 738-747. [[PDF](http://proceedings.mlr.press/v54/jerfel17a.html)]

ME Basbug, BE Engelhardt (2016). Hierarchical compound Poisson factorization. *Proceedings of the International Conference on Machine Learning (ICML)* 1795–1803. [[PDF](https://proceedings.mlr.press/v48/basbug16.html)]

C Gao, S Zhao, IC McDowell, CD Brown, BE Engelhardt (2016). Context-specific and differential gene co-expression networks via Bayesian biclustering models. *PLoS Computational Biology* 12:e1004791. [[PDF](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004791)]

PD Tonner, CD Darnell, BE Engelhardt, A Schmid (2016). Detecting differential growth of microbial populations with Gaussian process regression. *Genome Research* 27:320-333. [[PDF](https://genome.cshlp.org/content/27/2/320.long)]

S Zhao, C Gao, S Mukherjee, BE Engelhardt (2016). Bayesian group latent factor analysis with structured sparse priors. *Journal of Machine Learning Research (JMLR)* 17(196):1−47. [[PDF](https://jmlr.org/papers/v17/14-472.html)]

BE Engelhardt, CD Brown (2015). Diving deeper to predict noncoding sequence function. *Nature Methods (News & Views*; not peer-reviewed) 12(10):925–926. [[PDF](https://www.nature.com/articles/nmeth.3604)]

SM van Den Berg, MH de Moor, KJ Verweij, RF Krueger, M Luciano, AA Vasquez, LK Matteson, J Derringer, T Esko, N Amin, *et al.* (2015). Meta-analysis of genome-wide association studies for extraversion: Findings from the Genetics of Personality Consortium. *Behavior Genetics* 46(2):170–182. [[PDF](https://psycnet.apa.org/record/2015-42530-001)]

W Zhang, T Spector, P Deloukas, JT Bell, BE Engelhardt (2015). Predicting genome-wide DNA methylation using methylation marks, genomic position, and DNA regulatory elements. *Genome Biology*, 16(1):14. [[PDF](doi:10.1186/s13059-015-0581-9)]

D Mimno, DM Blei, BE Engelhardt (2015). Posterior predictive checks to quantify lack-of-fit in admixture models of latent population structure. *Proceedings of the National Academy of Sciences (PNAS)*, 112(26):E3341–50. [[PDF](doi:10.1073/pnas.1412301112)] [[Code](https://github.com/mimno/admixture-ppc)]

Genetics of Personality Consortium. (2015). Meta-analysis of genome-wide association studies for neuroticism, and the polygenic association with major depressive disorder. *JAMA Psychiatry*, 72(7):642-650. [[PDF](doi:10.1001/jamapsychiatry.2015.0554)]

AB Hart, ER Gamazon, BE Engelhardt, P Sklar, AK Kähler, CM Hultman, PF Sullivan, BM Neale, SV Faraone, H de Wit, NJ Cox, A Palmer (2014). Genetic variation associated with euphorigenic effects of *d*-amphetamine is associated with diminished risk for schizophrenia and attention deficit hyperactivity disorder. *Proceedings of the National Academy of Sciences (PNAS)*, 111(16):5968–73.
[[PDF](doi:10.1073/pnas.1318810111)]

LM Mangravite\*, BE Engelhardt\*, MW Medina, JD Smith, CD Brown, DI Chasman, BH Mecham, B Howie, H Shim, D Naidoo, Q Feng, MJ Rieder, Y Chen, JI Rotter, PM Ridker, JC Hopewell, S Parish, J Armitage, R Collins, RA Wilke, DA Nickerson, M Stephens, RM Krauss (2013). A statin-dependent QTL for *GATM* expression is associated with statin-induced myopathy. *Nature*, 502(7471):377–80. 
[[PDF](doi:10.1038/nature12508)]

F Mordelet, J Horton, AJ Hartemink, BE Engelhardt, R Gordân (2013). Stability selection for regression-based models of transcription factor-DNA binding specificity. *Bioinformatics*, 29(13):i117–25. [[PDF](doi:10.1093/bioinformatics/btt221)]

CD Brown, LM Mangravite, BE Engelhardt (2013). Integrative modeling of eQTLs and cis-regulatory elements suggests mechanisms underlying cell type specificity of eQTLs. *PLoS Genetics*, 9(8):e1003649. [[PDF](doi:10.1371/journal.pgen.1003649)]

KE Muratore, BE Engelhardt, JR Srouji, MI Jordan, SE Brenner, JF Kirsch (2013). Molecular function prediction for a family exhibiting evolutionary tendencies toward substrate specificity swapping: Recurrence of tyrosine aminotransferase activity in the I$\alpha$ subfamily. *Proteins: Structure, Function, and Bioinformatics*, 81(9):1593–1609.
[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3823064/pdf/prot0081-1593.pdf)]

AB Hart*, BE Engelhardt*, MC Wardle, G Sokoloff, M Stephens, H de Wit, A Palmer (2012). Genome-wide association study of *d*-amphetamine response in healthy volunteers identifies putative associations, including cadherin 13 (*CDH13*). *PLoS One*, 7(8):e42646. 
[[PDF](doi:10.1371/journal.pone.0042646)]

BE Engelhardt, MI Jordan, JR Srouji, SE Brenner (2011). Genome-scale phylogenetic function annotation of large and diverse protein families. *Genome Research*, 21(11):1969–80. 
[[PDF](doi:10.1101/gr.104687.109)]

BE Engelhardt, M Stephens (2010). Analysis of population structure: A unifying framework and novel methods based on sparse factor analysis. *PLoS Genetics*, 6(9):e1001117.
[[PDF](doi:10.1371/journal.pgen.1001117)]

JK Pickrell, JC Marioni, AA Pai, JF Degner, BE Engelhardt, E Nkadori, JB Veyrieras, M Stephens, Y Gilad, JK Pritchard (2010). Understanding mechanisms underlying human gene expression variation with RNA sequencing. *Nature*, 464(7289):768–772. 
[[PDF](doi:10.1038/nature08872)]

BE Engelhardt, MI Jordan, SE Brenner (2006). A graphical model for predicting protein molecular function. *Proceedings of the 23rd International Conference on Machine Learning (ICML)*, 297–304. 
[[PDF](10.1145/1143844.1143882)]

BE Engelhardt, MI Jordan, KE Muratore, SE Brenner (2005). Protein molecular function prediction by Bayesian phylogenomics. *PLoS Computational Biology*, 1(5), e45.
[[PDF](10.1371/journal.pcbi.0010045)]
