---
title: "Publications"
permalink: /publications/
date: 2022-06-11
---

## Preprints

A Jones, G Gundersen, BE Engelhardt (2022). Linking histology and molecular state across human tissues. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.05.31.494236)]

T Fitzgerald, A Jones, BE Engelhardt (2022). A Poisson reduced-rank regression model for association mapping in sequencing data. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.05.31.494236)] [[Code](https://github.com/tianafitz/PRRR)]

A Mandyam, A Jones, K Laudanski, BE Engelhardt. Nested policy reinforcement learning. [[arXiv](https://arxiv.org/abs/2110.02879)]

A Jones, FW Townes, D Li, BE Engelhardt (2022). Alignment of spatial genomics and histology data using deep Gaussian processes. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2022.01.10.475692)] [[Code](https://github.com/andrewcharlesjones/spatial-alignment)]

AD Gewirtz, FW Townes, BE Engelhardt (2021). Telescoping bimodal latent Dirichlet allocation to identify expression QTLs across tissues. [[bioRxiv](https://www.biorxiv.org/content/10.1101/2021.10.27.466156)] [[Code](https://github.com/gewirtz/TBLDA)]

FW Townes, BE Engelhardt. Nonnegative spatial factorization. [[arXiv](https://arxiv.org/abs/2110.06122)] [[Code](https://github.com/willtownes/nsf-paper)]

D Li*, A Jones*, BE Engelhardt. Probabilistic contrastive principal component analysis. [PDF] [[Code](https://github.com/andrewcharlesjones/pcpca)]

D Li*, A Jones*, S Banerjee, BE Engelhardt. Multi-group Gaussian processes. [[arXiv](https://arxiv.org/abs/2110.08411)] [[Code](https://github.com/andrewcharlesjones/multi-group-GP)]

IN Grabski, R de Vito, BE Engelhardt. Bayesian ordinal quantile regression with a partially collapsed Gibbs sampler. [PDF]

A Verma, BE Engelhardt. A Bayesian nonparametric semi-supervised model for integration of multiple single-cell experiments. [PDF] [[Code](https://github.com/architverma1/sc-manifold-alignment)]

de Vito, R., Grabski, I. N., Aguiar, D., Schneper, L. M., Verma, A., Castillo Fernandez, J., Mitchell, C., Bell, J., McLanahan, S., Notterman, D. A., & Engelhardt, B. E. Differentially methylated regions and methylation QTLs for teen depression and early puberty in the Fragile Families Child Wellbeing Study. [PDF]

B Dumitrascu, K Feng, BE Engelhardt. GT-TS: Experimental design for maximizing cell type discovery in single-cell data. [[bioRxiv](https://www.biorxiv.org/content/10.1101/386540)] [[Talk](https://www.youtube.com/watch?v=sDFe3e33m_I)]

AJB Chaney, A Verma, Y Lee, BE Engelhardt. Nonparametric deconvolution models. [[arXiv](arxiv.org/abs/2003.07718)] [[Code](https://github.com/ajbc/ndm)]

C Gao, CD Brown, BE Engelhardt. A latent factor model with a mixture of sparse and dense factors to model gene expression data affected by technical and biological covariates. [PDF]

## Publications

Martinet G, Strzalkowski A, Engelhardt BE (2022). Variance minimization in the Wasserstein space for invariant causal prediction. *Artificial Intelligence and Statistics (AISTATS)*. [PDF]

Jones, A., Townes, W. F., Li, D., & Engelhardt, B. E. (2022). Contrastive latent variable modeling with application to case-control sequencing experiments. *Annals of Applied Statistics (AOAS)*. [PDF] [[Code](https://github.com/andrewcharlesjones/cplvm)]

Cui, S., Yoo, E. C., Li, D., Laudanski, K., & Engelhardt, B. E.. (2022). Hierarchical Gaussian processes and mixtures of experts to model Covid-19 patient trajectories. *Proceedings of the Pacific Symposium on Biocomputing (PSB)*.
[PDF]

J Ash*, G Darnell*, D Munro*, BE Engelhardt (2021). Joint analysis of gene expression levels and histological images identifies genes associated with tissue morphology. *Nature Communications*, 12(1609).
[PDF] [[Code](https://github.com/daniel-munro/imageCCA)]

B Dumitrascu, S Villar, DG Mixon, BE Engelhardt (2021). Optimal marker gene selection for cell type discrimination in single cell analyses. *Nature Communications*, 12(1186).
[PDF] [[Code](https://github.com/solevillar/scGeneFit-python)]

Lu, J., Dumitrascu, B., McDowell, I. C., Barrera, A., K., H. L., Leichter, S. M., Reddy, T. E., & Engelhardt, B. E.. (2021). Causal network inference from gene transcriptional time series response to glucocorticoids. PLoS Computational Biology, 17(1), e1008223.
[PDF]

Gundersen, G. W., Cai, D., Zhou, C., Engelhardt, B. E., & Adams, R. P.. (2021). Active multi-fidelity Bayesian online changepoint detection. Uncertainty in Artifical Intelligence (UAI).
[PDF]

Wu, A., Nastase, S. A., Baldassano, C. A., Turk-Browne, N. B., Norman, K. A., Engelhardt, B. E., & Pillow, J. W.. (2021). Brain kernel: a new spatial covariance function for fMRI data. NeuroImage, 15(245), 118580.
[PDF]

Verma, A., Jena, S., Isakov, D. R., Aoki, K., Toettcher, J. E., & Engelhardt, B. E.. (2021). A self-exciting point process to study multi-cellular spatial signaling patterns. Proceedings of the National Academy of Sciences (PNAS), 118, e2026123118.
[PDF] [[Code](https://github.com/architverma1/CPP)]

Mandyam, A., Yoo, E. C., Soules, J., Laudanski, K., & Engelhardt, B. E.. (2021). COP-E-CAT: Cleaning and organization pipeline for EHR computational and analytic tasks. Proceedings of the 12th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics (BCB).
[PDF]

Gundersen, G. W., Zhang, M. M., & Engelhardt, B. E.. (2021). Latent variable modeling with random features. Artificial Intelligence and Statistics (AISTATS).
[PDF]

Prasad, N., Engelhardt, B. E., & Doshi-Velez, F.. (2020). Defining admissible rewards for high confidence policy evaluation. ACM Conference on Health, Inference, and Learning (CHIL).
[PDF]

Cheng, L., Dumitrascu, B., Zhang, M. M., Chivers, C., Draugelis, M. E., Li, K., & Engelhardt, B. E.. (2020). Patient-specific effects of medication using latent force models with Gaussian processes. Artificial Intelligence and Statistics (AISTATS).
[PDF]

Salganik, M., & others. (2020). Measuring the predictability of life outcomes with a scientific mass collaboration. Proceedings of the National Academy of Sciences (PNAS).
[PDF]

Cheng, L., Darnell, G., Chivers, C., Draugelis, M. E., Li, K., & Engelhardt, B. E.. (2020). Sparse multi-output Gaussian processes for medical time series prediction. BMC Medical Informatics and Decision Making, 20(152).
[PDF]

Camerlenghi*, F., Dumitrascu*, B., Ferrari, F., Engelhardt, B. E., & Favaro, S.. (2020). Nonparametric Bayesian multi-armed bandits for single cell experiment design. Annals of Applied Statistics (AOAS).
[PDF] [[Code]](https://github.com/fedfer/HPYsinglecell)

Oliva, M., GTEx Consortium, & others. (2020). The impact of sex on gene expression across human tissues. Science, 369. doi:10.1126/science.aba3066
[PDF]

GTEx Consortium. (2020). The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science, 369, 1318–1330. doi:10.1126/science.aaz1776
[PDF]

Gill, D., Arvanitis, M., Carter, P., Cordero, A. H. I., Jo, B., Karhunen, V., Larsson, S. C., Li, X., Lockhart, S. M., Mason, A., Pashos, E., Saha, A., Tan, V. Y., Zuber, V., Bossé, Y., Fahle, S., Hao, K., Jiang, T., Joubert, P., Lunt†, A. C., Ouwehand, W. H., Roberts, D. J., Timens, W., van den Berge, M., Watkins, N. A., Battle, A., Butterworth, A. S., Danesh, J., Angelantonio, E. D., Engelhardt, B. E., Peters, J. E., Sin, D. D., & Burgess, S.. (2020). ACE inhibition and cardiometabolic risk factors, lung ACE2 and TMPRSS2 gene expression, and plasma ACE2 levels: a Mendelian randomization study. Royal Society of Open Science, 7, 200958.
[PDF]

Verma, A., & Engelhardt, B. E. (2020). A robust nonlinear low-dimensional manifold for single cell RNA-seq data. BMC Bioinformatics, 21(324).
[PDF] [[Code]](https://github.com/architverma1/tGPLVM)

Elyanow, R., Dumitrascu, B., Engelhardt, B. E., & Raphael, B. J.. (2020). netNMF-sc: Leveraging gene-gene interactions for imputation and dimensionality reduction in single-cell expression analysis. Genome Research, 20(2), 195–204.
[PDF]

Cheng, L., Prasad, N., & Engelhardt, B. E.. (2019). An optimal policy for patient laboratory tests in intensive care units. Proceedings of the Pacific Symposium on Biocomputing (PSB).
[PDF]

Guan, G., & Engelhardt, B. E.. (2019). Predicting sick patient volume in a pediatric outpatient setting using time series analysis. Proceedings of Machine Learning for Health Care (MLHC).
[PDF]

Elyanow, R., Dumitrascu, B., Engelhardt, B. E., & Raphael, B. J.. (2019). netNMF-sc: Leveraging gene-gene interactions for imputation and dimensionality reduction in single-cell expression analysis. Proceedings of the 23rd International Conference on Research in Computational Molecular Biology (RECOMB).
[PDF]

Gundersen, G., Dumitrascu, B., & Engelhardt, B. E.. (2019). End-to-end training of deep probabilistic CCA on paired biomedical observations. in Proceedings of the Conference on Uncertainty in Artificial Intelligence (UAI).
[PDF]

McDowell, I. C., Manandhar, D., Vockley, C. M., Schmid, A., Reddy*, T. E., & Engelhardt*, B. E.. (2018). Clustering gene expression time series data using an infinite Gaussian process mixture model. PLoS Computational Biology, 14(1), e1005896.
[PDF]

Dumitrascu, B., Darnell, G., Ayroles, J., & Engelhardt, B. E.. (2018). A Bayesian test to identify variance effects. Bioinformatics, bty565.
[PDF] [[Code]](https://github.com/b2du/bth)

McDowell, I. C., Barrero, A., D’Ippolito, A. M., Vockley, C. M., Hong, L. K., Leichter, S. M., Bartelt, L. C., Majoros, W. H., Song, L., Safi, A., Koçak, D. D., Gersbach, C. A., Hartemink, A. J., Crawford, G. E., Engelhardt, B. E., & Reddy, T. E.. (2018). Glucocorticoid receptor recruits to enhancers and drives activation by motif-directed binding. Genome Research.
[PDF]

Aguiar, D., Cheng, L., Dumitrascu, B., Mordelet, F., Pai, A. A., & Engelhardt, B. E.. (2018). BIISQ: Bayesian nonparametric discovery of Isoforms and Individual Specific Quantification. Nature Communications, 9(1681).
[PDF]

Chaney, A. J. B., Stewart, B. M., & Engelhardt, B. E.. (2018). How algorithmic confounding in recommendation systems increases homogeneity and decreases utility. 12th ACM Conference on Recommender Systems.
[PDF]

Dumitrascu*, B., Feng*, K., & Engelhardt, B. E.. (2018). PG-TS: Improved Thompson sampling for logistic contextual bandits. Proceedings of Neural Information Processing Systems (NeurIPS).
[PDF]

Cohen, J. D., Daw, N., Engelhardt, B., Hasson, U., Li, K., Niv, Y., Norman, K. A., Pillow, J., Ramadge, P. J., Turk-Browne, N. B., & Willke, T. L.. (2017). Computational approaches to fMRI analysis. Nature Neuroscience, 20(3), 304–313.
[PDF]

Srivastava, S., Engelhardt, B. E., & Dunson, D. B.. (2017). Expandable factor analysis. Biometrika, asx030.
[PDF]

GTEx Consortium, Battle*, A., Brown*, C. D., Engelhardt*, B. E., & Montgomery*, S. M.. (2017). Genetic effects on gene expression across human tissues. Nature, 550, 204–213.
[PDF]

Zhao, S., Engelhardt, B. E., Mukherjee, S., & Dunson, D. B.. (2017). Fast moment estimation for generalized latent Dirichlet models. Journal of the American Statistical Association (JASA).
[PDF]

Darnell, G., Georgiev, S., Mukherjee, S., & Engelhardt, B. E.. (2017). Adaptive randomized dimension reduction on massive data. Journal of Machine Learning Research (JMLR), 18(140), 1-30.
[PDF]

Saha, A., Kim*, Y., Gewirtz*, A. D. H., Jo, B., Gao, C., McDowell, I. C., GTEx Consortium, Engelhardt*, B. E., & Battle*, A.. (2017). Co-expression networks reveal the tissue-specific regulation of transcription and splicing. Genome Research, 27(11), 1843–1858.
[PDF]

Prasad, N., Cheng, L., Chivers, C., Draugelis, M., & Engelhardt, B. E.. (2017). A reinforcement learning approach to weaning of mechanical ventilation in intensive care units. Proceedings of Uncertainty in Artificial Intelligence (UAI), 1–9.
[PDF]

Jerfel, G., Basbug, M. E., & Engelhardt, B. E.. (2017). Dynamic collaborative filtering with compound Poisson factorization. Proceedings of Artificial Intelligence and Statistics (AISTATS), 54, 738-747.
[PDF]

Basbug, M. E., & Engelhardt, B. E.. (2016). Hierarchical compound Poisson factorization. Proceedings of the International Conference on Machine Learning (ICML), 1795–1803.
[PDF]

Gao, C., Zhao, S., McDowell, I. C., Brown, C. D., & Engelhardt, B. E.. (2016). Context-specific and differential gene co-expression networks via Bayesian biclustering models. PLoS Computational Biology, 12, e1004791.
[PDF]

Tonner, P. D., Darnell, C. D., Engelhardt, B. E., & Schmid, A.. (2016). Detecting differential growth of microbial populations with Gaussian process regression. Genome Research, 27, 320-333.
[PDF]

Zhao, S., Gao, C., Mukherjee, S., & Engelhardt, B. E.. (2016). Bayesian group latent factor analysis with structured sparse priors. Journal of Machine Learning Research (JMLR).
[PDF]

Engelhardt, B. E., & Brown, C. D.. (2015). Diving deeper to predict noncoding sequence function. *Nature Methods (News & Views*; not peer-reviewed), 12(10), 925–926.
[PDF]

van Den Berg, S. M., de Moor, M. H., Verweij, K. J., Krueger, R. F., Luciano, M., Vasquez, A. A., Matteson, L. K., Derringer, J., Esko, T., Amin, N., & others. (2015). Meta-analysis of genome-wide association studies for extraversion: Findings from the Genetics of Personality Consortium. *Behavior Genetics*, 1–13. doi:10.1007/s10519-015-9735-5
[PDF]

Zhang, W., Spector, T., Deloukas, P., Bell, J., & Engelhardt, B. E.. (2015). Predicting genome-wide DNA methylation using methylation marks, genomic position, and DNA regulatory elements. *Genome Biology*, 16(1), 14. doi:10.1186/s13059-015-0581-9
[PDF]

Mimno, D., Blei, D. M., & Engelhardt, B. E.. (2015). Posterior predictive checks to quantify lack-of-fit in admixture models of latent population structure. *Proceedings of the National Academy of Sciences (PNAS)*, 112(26), E3341–50. doi:10.1073/pnas.1412301112
[PDF]

Genetics of Personality Consortium. (2015). Meta-analysis of genome-wide association studies for neuroticism, and the polygenic association with major depressive disorder. *JAMA Psychiatry*, 72(7), 642-650. doi:10.1001/jamapsychiatry.2015.0554
[PDF]

Hart, A. B., Gamazon, E. R., Engelhardt, B. E., Sklar, P., Kähler, A. K., Hultman, C. M., Sullivan, P. F., Neale, B. M., Faraone, S. V., de Wit, H., Cox, N. J., & a Palmer, A.. (2014). Genetic variation associated with euphorigenic effects of d-amphetamine is associated with diminished risk for schizophrenia and attention deficit hyperactivity disorder. *Proceedings of the National Academy of Sciences (PNAS)*, 111(16), 5968–73. doi:10.1073/pnas.1318810111
[PDF]

Mangravite*, L. M., Engelhardt*, B. E., Medina, M. W., Smith, J. D., Brown, C. D., Chasman, D. I., Mecham, B. H., Howie, B., Shim, H., Naidoo, D., Feng, Q., Rieder, M. J., Chen, Y. I., Rotter, J. I., Ridker, P. M., Hopewell, J. C., Parish, S., Armitage, J., Collins, R., Wilke, R. A., Nickerson, D. A., Stephens, M., & Krauss, R. M.. (2013). A statin-dependent QTL for GATM expression is associated with statin-induced myopathy. Nature, 502(7471), 377–80. doi:10.1038/nature12508
[PDF]

Mordelet, F., Horton, J., Hartemink, A. J., Engelhardt, B. E., & Gordân, R.. (2013). Stability selection for regression-based models of transcription factor-DNA binding specificity. Bioinformatics, 29(13), i117–25. doi:10.1093/bioinformatics/btt221
[PDF]

Brown, C. D., Mangravite, L. M., & Engelhardt, B. E.. (2013). Integrative modeling of eQTLs and cis-regulatory elements suggests mechanisms underlying cell type specificity of eQTLs. PLoS Genetics, 9(8), e1003649. doi:10.1371/journal.pgen.1003649
[PDF]

Muratore, K. E., Engelhardt, B. E., Srouji, J. R., Jordan, M. I., Brenner, S. E., & Kirsch, J. F.. (2013). Molecular function prediction for a family exhibiting evolutionary tendencies toward substrate specificity swapping: Recurrence of tyrosine aminotransferase activity in the I$\alpha$ subfamily. Proteins: Structure, Function, and Bioinformatics, 81(9), 1593–1609.
[PDF]

Hart*, A. B., Engelhardt*, B. E., Wardle, M. C., Sokoloff, G., Stephens, M., de Wit, H., & Palmer, A.. (2012). Genome-wide association study of d-amphetamine response in healthy volunteers identifies putative associations, including cadherin 13 (CDH13). PLoS One, 7(8), e42646. doi:10.1371/journal.pone.0042646
[PDF]

Engelhardt, B. E., Jordan, M. I., Srouji, J. R., & Brenner, S. E.. (2011). Genome-scale phylogenetic function annotation of large and diverse protein families. Genome Research, 21(11), 1969–80. doi:10.1101/gr.104687.109
[PDF]

Engelhardt, B. E., & Stephens, M. (2010). Analysis of population structure: A unifying framework and novel methods based on sparse factor analysis. PLoS Genetics, 6(9), e1001117. doi:10.1371/journal.pgen.1001117
[PDF]

Pickrell, J. K., Marioni, J. C., Pai, A. A., Degner, J. F., Engelhardt, B. E., Nkadori, E., Veyrieras, J., Stephens, M., Gilad, Y., & Pritchard, J. K.. (2010). Understanding mechanisms underlying human gene expression variation with RNA sequencing. Nature, 464(7289), 768–772. doi:10.1038/nature08872
[PDF]

Engelhardt, B. E., Jordan, M. I., & Brenner, S. E.. (2006). A graphical model for predicting protein molecular function. Proceedings of the 23rd International Conference on Machine Learning (ICML), 297–304. doi:10.1145/1143844.1143882
[PDF]

Engelhardt, B. E., Jordan, M. I., Muratore, K. E., & Brenner, S. E. (2005). Protein molecular function prediction by Bayesian phylogenomics. PLoS Computational Biology, 1(5), e45. doi:10.1371/journal.pcbi.0010045 
[PDF]
